cairo/tests/ethereum/cancun/test_state.py

```diff
@@ -0,0 +1,69 @@
+import pytest
+from ethereum_types.bytes import Bytes32
+from hypothesis import given
+from hypothesis import strategies as st
+
+from ethereum.cancun.fork_types import Address
+from ethereum.cancun.state import State, get_account, get_account_optional, get_storage
+from tests.utils.strategies import address, bytes32, state
+
+pytestmark = pytest.mark.python_vm
+
+
+class TestState:
+    @given(state=state, address=address, draw_address_from_state=st.booleans())
+    def test_get_account(
+        self, cairo_run, state: State, address: Address, draw_address_from_state: bool
+    ):
+        if draw_address_from_state and state._main_trie._data.keys():
```

Comment:
```suggestion
        if draw_address_from_state and len(state._main_trie._data):
```

Reply:
yes, if works, was thinking it is easier to read 
but fine for me also to only have len(state._main_trie._data) withtout the comparaison to 0